From 75acd325bd42539334f4d2ae0152cb70cc5e8311 Mon Sep 17 00:00:00 2001
From: Nicolas Fella <nicolas.fella@gmx.de>
Date: Mon, 22 Aug 2022 21:06:15 +0200
Subject: [PATCH] Don't search for KF5Activities on Windows/Mac

Activities don't make sense there

(cherry picked from commit a1a08e1c19fa52e5ee669614948023447e38eec4)
---
 apps/lib/CMakeLists.txt | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/apps/lib/CMakeLists.txt b/apps/lib/CMakeLists.txt
index a883cf5c5..6ba178aa6 100644
--- a/apps/lib/CMakeLists.txt
+++ b/apps/lib/CMakeLists.txt
@@ -34,10 +34,12 @@ find_package(
     WindowSystem
     DBusAddons
     Crash
-  OPTIONAL_COMPONENTS
-    Activities
 )
 
+if (NOT WIN32 AND NOT APPLE)
+    find_package(KF5 ${KF5_DEP_VERSION} OPTIONAL_COMPONENTS Activities)
+endif()
+
 target_link_libraries(
   kateprivate
   PUBLIC
-- 
2.37.2

